superset/assets/src/explore/store.js

```diff
@@ -104,13 +104,6 @@ export function applyDefaultFormData(form_data) {
       formData[k] = form_data[k];
     }
   });
-  // fill in additional params stored in form_data but not used by control
-  Object.keys(form_data)
```

Comment:
Doing forensics here: @graceguo-supercat added this as part of `[Explore] Save custom url parameters when user save slices (#4578)`